plugins/nisq_analyzer.py

```diff
@@ -86,6 +147,35 @@ def __init__(self, app: Optional[Flask]) -> None:
 
         self.url = app.config.get("NISQ_ANALYZER_UI_URL")
         self.url = environ.get("NISQ_ANALYZER_UI_URL", self.url)
+        self.url += "/#/algorithms"
```

Comment:
Use <https://docs.python.org/3/library/urllib.parse.html#module-urllib.parse> to work with and manipulate URLs.
Also, do not use the URL as is in the entryPoint.href but add the query parameter (that is currently added by the QHAna UI in the related PR) in the pluginsView. (see other comment.